packages/generator-open-wc/README.md

```diff
@@ -59,7 +47,7 @@ yo open-wc:scaffold-vanilla
 
 ### Available scaffold generators:
 
-- `yo open-wc:scaffold-vanilla`<br/> 
+- `scaffold-vanilla`<br/> 
```

Comment:
is gone... "replaced" with `wc-lit-element` but it only scaffolds webcomponent and does not install everything... 
it seem the more prefered way of working is to make a small thing (without much) play around and once you feel ready you add testing/demoing/building/...